dspace-server-webapp/src/main/java/org/dspace/app/rest/RestResourceController.java

```diff
@@ -895,6 +895,17 @@ public RepresentationModel findRel(HttpServletRequest request, HttpServletRespon
 
     }
 
+    private Page<? extends RestModel> getRestModelsWithoutNullValues(Pageable page,
```

Comment:
It appears the changes in `RestResourceController` are unnecessary.  This `private` method is never used, and the only other changes are space-related. 

Can we simply undo all the changes to this controller?

Reply:
The changes have been removed